ts3.4/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Why do we need this file? It's not mentioned in the downlevel-dts instructions.

Reply:
See step #4 of https://github.com/sandersn/downlevel-dts#usage

Reply:
Oops. I missed that.

Do we want to check this copy in though?

Reply:
Good point, done